gcloud-java-dns/src/main/java/com/google/gcloud/dns/spi/DefaultDnsRpc.java

```diff
@@ -61,7 +61,7 @@ public ManagedZone create(ManagedZone zone, Map<Option, ?> options) throws DnsEx
           .setFields(FIELDS.getString(options))
           .execute();
     } catch (IOException ex) {
-      throw translate(ex);
+      throw translate(ex, false);
```

Comment:
see @garrettjonesgoogle comment, which I partially agree with.

I think that means there is some more work to do, however for now I suggest to keep this as `true` and create a separate issue suggesting that some idempotent method may report a wrong exception when retried (such as in this case 'already exists'). 

The reason I think it is OK to retry it because the implication is just misreported error (e.g. 'already exists' instead of 500 or timeout). In either case the user will get an exception.